CHANGELOG.md

```diff
@@ -20,13 +20,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fix disabling of send streams when local video was not enabled by integrating empty encoder params into `VideoStreamIndex` when sending is disabled.
 - Fix `visibilitychange` typo in `InMemoryJSONEventBuffer`.
+- Take into account portrait video dimension when calculating resolution scaling for nscale video bandwidth uplink 
+  policy.
   
 ### Changed
 
 - Ignore `enableUnifiedPlanForChromiumBasedBrowsers` value (i.e. treat as always equaling the current default value of `true`) in `MeetingSesstionConfiguration`.  Chrome is [in the processing](https://groups.google.com/g/discuss-webrtc/c/UBtZfawdIAA/m/m-4wnVHXBgAJ) of deprecating and removing Plan-B which would cause breakage in applications still trying to use it.  This will have no effect on SDK behavior` and has been the default since 1.17.0.
 - Change `appVersionName` and `appVersionCode` fields to `appName` and `appVersion` respectively.
 - Update similar log messages in `DefaultMessagingSession` and `DefaultSignalingClient`.
 - Change the serverless demo's `deploy.js` script to rebuild the demo on each run. This should remove a manual step of rebuilding the demo.
+- Disable nscale resolution scaling for Android device due to Android H.264 encoding.
```

Comment:
Should we expand on the issue a bit here, or tag any existing issues for clarity?

Reply:
Added info in PR description and we have not been adding issue tags in changelogs.